acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ThreadPantheonNodeRunner.java

```diff
@@ -59,7 +59,8 @@ public void startNode(final PantheonNode node) {
               ethNetworkConfig,
               false,
               node.getMiningParameters(),
-              true);
+              true,
+              node.homeDirectory().resolve("key").toFile());
```

Comment:
Can we avoid the duplication of homeDirectory.resolve("key").toFile() here, in PantheonNode and in PantheonCommand somehow?  Maybe  we need `KeyPairUtil.getDefaultKeyFile(homeDirectory)`?

Reply:
Ok, I will do that.